### PR TITLE
Remove metric state and resources that will be migrated to that repo

### PR DESCRIPTION
**Pull request by steveburton4**
Remove metric state and resources that will be migrated to that repo

### CODE REVIEW

File: create-workspace.yml [hunk 1]
```diff
@@ -3,8 +3,8 @@ platform: linux
 image_resource:
   type: docker-image
   source:
-    repository: ((dataworks.terraform_repository))
-    tag: ((dataworks.terraform_version))
+    repository: hashicorp/terraform
+    tag: 0.12.31
```
Comment (ChrisScottThomas) [inline create-workspace.yml]:
```suggestion
    repository: ((dataworks.terraform_repository))
    tag: ((dataworks.terraform_12_version))
```
Does this not work?

File: modules/cognito/outputs.tf [hunk 2]
```diff
@@ -7,5 +7,6 @@ output outputs {
     user_pool             = aws_cognito_user_pool.concourse
     user_pool_main_domain = aws_cognito_user_pool_domain.main
     issuer                = "https://cognito-idp.${local.region}.amazonaws.com/${aws_cognito_user_pool.concourse.id}"
+    concourse_pool        = aws_cognito_user_pool.concourse
```
Comment (ChrisScottThomas) [inline modules/cognito/outputs.tf]: What does this do?  same as `user_pool` ?

File: create-workspace.yml (hunk 1 above)
Comment (steveburton4) [reply create-workspace.yml]: Fixed this.

Comment (steveburton4) [reply create-workspace.yml]: It was when I tested a repo with that version and the docker image wasn't present

File: modules/cognito/outputs.tf (hunk 2 above)
Comment (steveburton4) [reply modules/cognito/outputs.tf]: This is needed for metrics repo to have the resource added

Comment (steveburton4) [reply modules/cognito/outputs.tf]: Oh I see. Yes removed.

**Review by ChrisScottThomas — DISMISSED** [review]

**Review by Mark0930 — APPROVED** [review]

**Pull request merged**